mappings/net/minecraft/world/loot/condition/BlockStatePropertyLootCondition.mapping

```diff
@@ -1,5 +1,38 @@
 CLASS cqn net/minecraft/world/loot/condition/BlockStatePropertyLootCondition
+	CLASS cqn$a Builder
+		FIELD a block Lbmb;
+		FIELD b availableProperties Ljava/util/Set;
+		FIELD c propertyValues Ljava/util/Map;
+		METHOD <init> (Lbmb;)V
+			ARG 1 block
+		METHOD a add (Lbwc;Ljava/lang/Comparable;)Lcqn$a;
+			ARG 1 property
+			ARG 2 value
 	CLASS cqn$b Factory
+		METHOD a getPropertyValueString (Lbwc;Ljava/lang/Object;)Ljava/lang/String;
+			ARG 0 property
+			ARG 1 value
+		METHOD a (Lcom/google/gson/JsonObject;Lbwc;Ljava/lang/Object;)V
+			ARG 1 property
+			ARG 2 value
 	FIELD a block Lbmb;
 	FIELD b properties Ljava/util/Map;
 	FIELD c predicate Ljava/util/function/Predicate;
+	METHOD <init> (Lbmb;Ljava/util/Map;)V
+		ARG 1 block
+		ARG 2 properties
+	METHOD a create (Lbmb;)Lcqn$a;
```

Comment:
There are many instances of this create method, both in loot conditions and loot functions. I hzave used `create` because some of them already used that mapping.
However, most of these methods return a `.Builder` of the condition/function. So I think something like `createBuilder` or `getBuilder` would be more appropiate here.

Reply:
Just `builder`